build.gradle

```diff
@@ -37,21 +37,21 @@ dependencies {
 	implementation group: 'javax.validation', name: 'validation-api', version: '2.0.1.Final'
 	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc'
 	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
-	implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.12'
+	runtimeOnly group: 'org.postgresql', name: 'postgresql', version: '42.2.14'
```

Comment:
This matches what the [spring initializr](https://start.spring.io/#!type=gradle-project&language=java&platformVersion=2.3.1.RELEASE&packaging=jar&jvmVersion=1.8&groupId=bio.terra.workspace&artifactId=terra-workspace-manager&name=terra-workspace-manager&description=&packageName=bio.terra.workspace.terra-workspace-manager&dependencies=web,data-jdbc,postgresql,liquibase) generates.